ipams/remote/remote.go

```diff
@@ -49,6 +55,14 @@ func (a *allocator) call(methodName string, arg interface{}, retVal PluginRespon
 	return nil
 }
 
+func (a *allocator) getCapabilities() (*ipamapi.Capability, error) {
+	var res api.GetCapabilityResponse
+	if err := a.call("GetCapabilities", nil, &res); err != nil {
```

Comment:
Will this fail for existing remote IPAM drivers which doesn't implement `GetCapabilities` ? :-1: 
We should ignore the error and return an empty capability.